cmd/up.go

```diff
@@ -54,6 +56,7 @@ var upCmd = &cobra.Command{
 }
 
 func init() {
+	upCmd.Flags().BoolVar(&UpInsecureRepo, "insecure-repository", false, "Use Insecure Repository")
```

Comment:
@surajnarwade why as global? 
This flag make sense only for `up` and `convert` and only for openshift.